src/service/viewer-impl.js

```diff
@@ -826,13 +829,17 @@ export class Viewer {
   }
 
   /**
+   * This message queues up the message to be sent when communication channel
+   * is established. If the communication channel is not established at
+   * this time, this method responds immediately with a Promise that yields
+   * `undefined` value.
    * @param {string} eventType
    * @param {*} data
    * @param {boolean} awaitResponse
    * @return {!Promise<*>|undefined}
    * @private
    */
-  sendMessage_(eventType, data, awaitResponse) {
+  sendMessageUnsafe_(eventType, data, awaitResponse) {
```

Comment:
What exactly is not guaranteed? That we send at all?

I think Unreliable is good then.